core/admin/mailu/sso/views/base.py

```diff
@@ -96,10 +97,12 @@ def _proxy():
     proxy_ip = flask.request.headers.get('X-Forwarded-By', flask.request.remote_addr)
     ip = ipaddress.ip_address(proxy_ip)
     if not any(ip in cidr for cidr in app.config['PROXY_AUTH_WHITELIST']):
+        flask.current_app.logger.error('Login failed by proxy - not on whitelist: from {client_ip} through {flask.request.remote_addr}.')
```

Comment:
I think there is the `f""` missing in the four log messages, isn't it?

Reply:
Good catch. that's bd398613